### PR TITLE
big chop of words

### PR DESCRIPTION
**Pull request by nrosed**
I did a big cut here, so please add back in or finesse as you see fit.
572 words

### CODE REVIEW

File: content/01.main_content.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+Ideally, audits would only use self-reported gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
+  While this is possible prospectively, surveying is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
+  In contrast, computationally-derived predictions allow for large-scale audits.
+  Numerous tools exist to algorithmically infer gender, nationality, and ethnicity using the feature likely to be present in most datasets: an individual's name. [@doi:10.1093/pan/mpw001; @http://genderize.io/]
+  These models are highly scalable, allowing auditors to broadly define the scope of their audit.
   
-Prediction models are not a panacea; several factors limit both their accuracy at recapitulating self-reported information and their ability to address the underlying motivating questions of diversity audits.
+Prediction models are not a panacea; several factors limit both their accuracy and utility.
```
Comment (arielah) [inline content/01.main_content.md:24]: I'm sad to see this distinction go but I agree it must be done for the word count. My kingdom for a couple extra hundred words to this article...

File: content/01.main_content.md [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
   For instance, gender associations of a given name can vary by culture, potentially biasing gender predictions where additional information is not available [@https://aclanthology.org/U14-1021].
   Also, most gender prediction models are trained on binary gender labels, which occludes assessing the representation of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
  
-Proxy predictions of ethnicity via name origin are more difficult still; choosing categories to probabilistically predict on is non-trivial and difficult to define. 
+Proxy predictions of ethnicity via name origin are more difficult still; defining categories needed for computational analyses is non-trivial and difficult to define.
```
Comment (arielah) [inline content/01.main_content.md]: I'm not sure this part is communicating the same information anymore. What prompted the change?

File: content/01.main_content.md [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
   Furthermore, there is no one-to-one mapping between having a name from a linguistic group and belonging to a minoritized or underrepresented group.
   Colonialism, immigration, and structural racism have affected most groups' linguistic history and inclusion or exclusion from scientific communities in complex ways that are nearly impossible to parse from names alone. 
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007]. 
-  If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively computational analysis.
+  <!-- If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively computational analysis. -->
```
Comment (arielah) [inline content/01.main_content.md:32]: I was thinking of chopping this too, so I agree it's the right call.

**Review by arielah — COMMENTED** [review]
This looks good! I'm amazed at how many words you were able to trim back. Just one sentence that I'm confused about.

File: content/01.main_content.md (hunk 1 above)
Comment (nrosed) [reply content/01.main_content.md:24]: 550 is too short! it requires loss of information :(

File: content/01.main_content.md (hunk 2 above)
Comment (nrosed) [reply content/01.main_content.md]: ah, yes... i think its better before actually. I was rewording it a lot, but ended up with almost the same number of words. I will revert.

Comment (nrosed) [reply content/01.main_content.md]: reverted in https://github.com/greenelab/disparities_opinion/pull/10/commits/aa299fe5b1ac6c57f203ce5386fca40aec0e9510

**Review by arielah — APPROVED** [review]

**Pull request merged**